index.html

```diff
@@ -295,8 +294,7 @@ <h3>
             </td>
             <td>
               A <a>list</a> of {{WakeLockSentinel}} objects, representing
-              active wake locks associated with the [=environment settings
-              object / responsible document=].
+              active wake locks associated with the <a>platform wake lock</a>.
```

Comment:
Would it make sense to use `{{Document}}` here instead of "platform wake lock"?

BTW, reading these bits again it feels like we're using the concept of "platform wake lock" incorrectly, and we should just say "a Document has the following internal slots". Is extending `Document` and other interfaces with new internal slots acceptable? I can take care of that separately if so.

Reply:
Yes, adding new internal slots (or, equivalently and a bit more preferred, associated values) is fine.

I couldn't really tell what these lists were associated with. The sentence above says "Each platform wake lock has an..." which is why I went with platform wake lock instead of Document.

Reply:
> I couldn't really tell what these lists were associated with. The sentence above says "Each platform wake lock has an..." which is why I went with platform wake lock instead of Document.

Yeah, I got a bit confused when reading this section again. The idea is essentially that a Document has one or more `WakeLockSentinel`s, and when list is emptied any existing wake lock backend is notified that the OS-side of the lock should be released.

Reply:
OK, I changed the description, although a separate cleanup would probably be good :)